ros/angel_system_nodes/angel_system_nodes/hl2ss_ros_bridge.py

```diff
@@ -22,6 +22,8 @@
     HeadsetAudioData,
     HeadsetPoseData,
     SpatialMesh,
+    # TODO: Is a message for depth AHAT needed here? Or from sensor_msgs?
+    # Currently re-using Image message
```

Comment:
If this is addressed can we remove the comment?

Reply:
A hint at this is retained by a lower comment pondering if other elements of the message received from HL2SS should be propagated down stream that we can determine at a later time.